scripts/check-plugin-meta.js

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env node
+
+const fs = require('fs')
```

Comment:
Please use `import` here.

Reply:
If we do that then we need to have `esm` as a dev dependency and you need to run the script with `node -r esm`

Reply:
That's a point.

Reply:
Damn, I just committed those changes. Adding esm as dev dependency shouldnt really be an issue as nuxt also uses it so we already have it anyway.

Reply:
Also now I can import the startCase util from src and dont have to copy that implementation :)

Reply:
Ok. I'am fine with that :)